docs/layers/software-delivery/eks-argocd/eks-argocd.mdx

```diff
@@ -53,6 +33,8 @@ For more information, please see:
   > should already be patched if you use Ruby or JavaScript and update your dependencies regularly. And if you use .NET
   > or Java, there’s probably nothing to worry about.
 
+</Note>
```

Comment:
I don't prefer such huge notes. 
```suggestion
```

Reply:
removed and used #### title (so that it doesnt show up on right sidebar)